vllm/model_executor/layers/quantization/gptq_hpu.py

```diff
@@ -0,0 +1,291 @@
+from fractions import Fraction
```

Comment:
@kzawora-intel This should probably go to vllm-hpu-extension what do you think?

Reply:
agreed

Reply:
agreed

Reply:
CUDA class file https://github.com/HabanaAI/vllm-fork/blob/2ea889afdac9df62190de14113e00e293d8e3aff/vllm/model_executor/layers/quantization/gptq.py also has this import.